lib/brains/tests/test_webhook.py

```diff
@@ -0,0 +1,55 @@
+from django.core.urlresolvers import reverse
+
+from braintree.webhook_notification import WebhookNotification
+from braintree.webhook_notification_gateway import WebhookNotificationGateway
+from nose.tools import eq_
+
+from lib.brains.tests.base import BraintreeTest
+
+
+def parsed(**kwargs):
+    data = {'kind': 'test', 'subject': ''}
+    data.update(kwargs)
+    return WebhookNotification(None, data)
+
+
+def example(**kwargs):
+    data = {'bt_signature': 'signature', 'bt_payload': 'payload'}
+    data.update(kwargs)
+    return data
+
+
+class TestWebhookVerify(BraintreeTest):
+    gateways = {'notify': WebhookNotificationGateway}
+
+    def setUp(self):
+        super(TestWebhookVerify, self).setUp()
+        self.url = reverse('braintree:webhook.verify')
+
+    def test_allowed(self):
+        self.allowed_verbs(self.url, ['get'])
+
+    def test_missing(self):
+        eq_(self.client.get(self.url).status_code, 422)
+
+    def test_ok(self):
+        self.mocks['notify'].verify.return_value = 'foo'
+        eq_(self.client.get(self.url + '?bt_challenge=x').json, 'foo')
```

Comment:
in cases like this I think it's helpful to add an assertion that the correct call was made, like ``self.mocks['notify'].verify.assert_called_with('x')``.